src/engraving/engravingmodule.cpp

```diff
@@ -262,6 +263,7 @@ void EngravingModule::onInit(const IApplication::RunMode& mode)
 
     Drumset::initDrumset();
     FiguredBass::readConfigFile(String());
+    FretDiagram::readHarmonyToDiagramFile("://data/harmony_to_diagram.xml");
```

Comment:
Perhaps we should do some sort of lazy reading: read it only when we have to. Because many users won't need this feature, but we will increase the loading time for everyone